spec/factories/shelter.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :shelter do
+    sequence(:name) {|n| "Shelter #{n}" }
+    sequence(:address) {|n| "#{n} Shelter Ave" }
+    sequence(:city) {|n| "#{n} City" }
+    sequence(:state) {|n| "#{n} State" }
+    sequence(:zip) {|n| "#{n} Zip" }
+  end
+end
```

Comment:
This creates new Shelter objects by inserting a number in all spots where n is. Each new object increments up by one...ex("Shelter 1", "Shelter 2")